workspace/src/contract.rs

```diff
@@ -440,11 +439,9 @@ impl<U> EvmAccount<U> {
         &self,
         account_id: A,
     ) -> Result<ViewResultDetails<u128>> {
-        let args = serde_json::to_string(&account_id.as_ref())?;
-        ViewResultDetails::try_from(
-            self.near_view(&View::FtBalanceOf, args.as_bytes().to_vec())
-                .await?,
-        )
+        let account = AccountId::from_str(account_id.as_ref()).unwrap();
+        let args = borsh::to_vec(&account).unwrap();
```

Comment:
I've changed this function in my PR, which also uses borsh but in a different way. But it's weird because I think my changes are already incorporated in this PR? Maybe needs another rebase.

Reply:
I could change this while made a merge.